flow/probes/ebpf.go

```diff
@@ -94,12 +101,59 @@ func tcpFlagTime(currFlagTime C.__u64, startKTimeNs int64, start time.Time) int6
 	return common.UnixMillis(start.Add(time.Duration(int64(currFlagTime) - startKTimeNs)))
 }
 
-func (p *EBPFProbe) newFlowOperation(ebpfFlow *EBPFFlow, kernFlow *C.struct_flow, startKTimeNs int64, start time.Time) *flow.Operation {
+func kernLayersPath(kernFlow *C.struct_flow) (layersPath string, hasGRE bool) {
+	hasGRE = false
+	notFirst := false
+	layersPath = ""
+	path := uint64(kernFlow.layers_path)
+	for i := C.LAYERS_PATH_LEN - 1; i >= 0; i-- {
+		layer := (path & (uint64(C.LAYERS_PATH_MASK) << uint(i*C.LAYERS_PATH_SHIFT))) >> uint(i*C.LAYERS_PATH_SHIFT)
+		if layer == 0 {
+			continue
+		}
+		if notFirst {
+			layersPath += "/"
+		}
+		notFirst = true
+		switch layer {
+		case C.ETH_LAYER:
+			layersPath += "Ethernet"
+		case C.ARP_LAYER:
+			layersPath += "Arp"
```

Comment:
ARP

We should use the gopacket string definition otherwise we will introduce difference between layer spelling (lower case)

Reply:
good catch